python-package/lightgbm/sklearn.py

```diff
@@ -578,21 +578,32 @@ def fit(self, X, y,
             feature_name='auto', categorical_feature='auto',
             callbacks=None, init_model=None):
         """Docstring is set after definition, using a template."""
+        params = self.get_params()
+
+        params.pop('objective', None)
+        for alias in _ConfigAliases.get('objective'):
+            if alias in params:
+                self._objective = params.pop(alias)
+                _log_warning(f"Found `{alias}` in params. Will use it instead of argument")
```

Comment:
Is this sentence complete? Should it be `Will use it instead of argument `objective``?

Reply:
In this particular case I agree with you. Warning message was improved in 2d0392ef6c5e5a8670dcc22684c63afe9faaa2ba.

However, please note that in this similar warning
https://github.com/microsoft/LightGBM/blob/da98f24711a2faab17f94e5b2a636e6609c93fa6/python-package/lightgbm/engine.py#L571-L574

we cannot add specific argument name which is being overwritten because it depends on whether user uses `train()` function directly or indirectly from the sklearn-wrapper. Please consider the following example:
```python
import lightgbm as lgb

from sklearn.datasets import load_boston

X, y = load_boston(return_X_y=True)
lgb_train = lgb.Dataset(X, y)

lgb.train({'n_iter': 5}, lgb_train)  # here n_iter is used instead of num_boost_round argument

lgb.LGBMRegressor(n_iter=5).fit(X, y)  # here n_iter is used instead of n_estimators argument
```